src/components/Clickable/Clickable.tsx

```diff
@@ -0,0 +1,30 @@
+import clsx from 'clsx'
+import {FC} from 'react'
+
+import styles from './Clickable.module.scss'
+
+interface ButtonProps {
+  onClick?: () => void
+  active?: boolean
+}
+
+export const Button: FC<ButtonProps> = ({children, onClick, active}) => {
+  return (
+    <span onClick={onClick} className={clsx(styles.actionButton, active && styles.disabled)}>
```

Comment:
to `active && styled.disabled` bolo take divne... jednak to nikde nepouzivame, a nedavalo uplne zmysel. som premenoval `active` na `disabled` a mozeme potom niekde pouzit

Reply:
takisto podla HTML by toto mal byt semanticky skor `<button>` ako `<span>`, pozriem sa ale, ako to vyzera so stylmi, pripadne to zmenim

Reply:
mimo tohto ako jednoduche komponenty je to fajn